client/src/containers/PlaceList/PlaceList.js

```
@@ -17,7 +18,7 @@ class PlacesList extends React.Component {
                     places: data
                 });
             })
-            .catch((err) => { })    
+            .catch((err) => { })
```

remove the catch - don't swallow errors like that, it's better that the application crashes (and fix issues) than it fails silently.